salt/grains/core.py

```diff
@@ -2953,10 +2953,10 @@ def _hw_data(osdata):
                 grains["uuid"] = res.group(1).strip().replace("'", "")
                 break
 
-        for regex in manufacture_regexes:
+        for regex in manufacturer_regexes:
             res = regex.search(data)
             if res and len(res.groups()) >= 1:
-                grains["manufacture"] = res.group(1).strip().replace("'", "")
+                grains["manufacturer"] = res.group(1).strip().replace("'", "")
```

Comment:
As per @OrangeDog and PR https://github.com/saltstack/salt/pull/53505
Need to keep the grain 'manufacture' around and mark it for deprecation, but instead of 'Magnesium' , mark it for Sulfur (American English spelling instead of English 'Sulphur'), also replace the misspelled verion (should be version)
```
                grains['manufacture'] = grains['manufacturer']  # Remove in Magnesium
                salt.utils.versions.warn_until('Magnesium',
                                               'manufacture has been replaced by manufacturer '
                                               'and will be removed in verion {version}')
```